packages/ui/components/form/inputs/Label.tsx

```diff
@@ -4,7 +4,11 @@ export function Label(props: JSX.IntrinsicElements["label"]) {
   return (
     <label
       {...props}
-      className={classNames("mb-2 block text-sm font-medium leading-none text-gray-700", props.className)}>
+      // TODO: Test labels inside the app when dark theme is enabled as app doesn't support dark theme
+      className={classNames(
+        "mb-2 block text-sm font-medium leading-none text-gray-700 dark:text-white",
+        props.className
+      )}>
```

Comment:
Youre all good here we have a constant

`isThemeSupported` which is only enabled on booking pages - else we only allow light mode :)

Reply:
Thanks @sean. I tested as well. Removing the comment